### PR TITLE
fix shellcheck failures in hack/test-integration.sh and hack/verify-t…

### PR DESCRIPTION
**Pull request by xichengliudui**
…est-featuregates.sh

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by xichengliudui** [comment]
I'm sorry that my previous submission was confused, so I had to create a new pr.
ref: #80740

File: hack/test-integration.sh [hunk 1]
```diff
@@ -20,8 +20,6 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
```
Comment (xichengliudui) [inline hack/test-integration.sh]: I found that KUBE_ROOT was not used in the script, so I removed it.
ref: https://github.com/koalaman/shellcheck/wiki/SC2034

**Review by BenTheElder — APPROVED** [review]
/lgtm
/approve
thanks!

**Pull request merged**